package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "integration-ts-example",
+    "type": "module",
+    "version": "1.0.0",
+    "description": "This example demonstrates how to use the Integration Node.js library.",
+    "main": "dist/light.js",
+    "dependencies": {
+        
+    },
+    "devDependencies": {
+		"@ava/typescript": "^5.0.0",
+		"@types/debug": "^4.1.12",
+		"@types/node": "^22.5.5",
+		"@types/ws": "^8.5.12",
+		"@typescript-eslint/eslint-plugin": "^8.8.1",
+		"@typescript-eslint/parser": "^8.8.1",
+		"ava": "^6.1.3",
+		"copyfiles": "^2.4.1",
+		"eslint": "^8.57.1",
+		"eslint-config-prettier": "^9.1.0",
+		"eslint-config-semistandard": "^17.0.0",
+		"eslint-plugin-prettier": "^5.2.1",
+		"eslint-plugin-promise": "^6.6.0",
+		"prettier": "^3.3.3",
+		"ts-node": "^10.9.2",
+		"typescript": "5.5.x"
+    },
+    "scripts": {
+        "format": "prettier --write \"./src/**/*.ts\"",
+        "lint": "prettier --check \"./src/**/*.ts\" && eslint \"./src/**/*.ts\" --fix",
```

Comment:
Use the same scripts `format`, `lint` and `code-check` as in your integration-node-library PR: https://github.com/ZsakaiBalint/integration-node-library/blob/iss-27/package.json#L38-L40